group-card/group-card.js

```diff
@@ -10,7 +10,7 @@ class GroupCard extends HTMLElement {
     if (!cardConfig.card) cardConfig.card = {};
     if (!cardConfig.card.type) cardConfig.card.type = 'entities';
 
-    const element = document.createElement(`hui-${cardConfig.card.type}-card`);
+    const element = document.createElement(`hui-${cardConfig.card.type.startsWith("custom:") ? cardConfig.card.type.substr("custom:".length) : cardConfig.card.type}-card`);
```

Comment:
This is pretty unreadable. Maybe expand the if and use a new const to create element

Reply:
Not sure which custom-card people would use. I tried to find one that has 'entities' but none have.